js/main.js

```diff
@@ -222,21 +222,22 @@ var mapView = {
 
   },
   initMap: function() {
-    var self = this;
+    var self = this,
+      mapStyle1 = [ 
+        { "featureType": "road", "elementType": "geometry.fill", "stylers": [ { "color": "#4f9f92" }, { "visibility": "on" } ] },
+        { "featureType": "water", "elementType": "geometry.stroke", "stylers": [ { "color": "#feff95" }, { "visibility": "on" }, { "weight": 1.2 } ] },
+        { "featureType": "landscape", "elementType": "geometry", "stylers": [ { "color": "#adff9d" }, { "visibility": "on" } ] },
+        { "featureType": "water", "stylers": [ { "visibility": "on" }, { "color": "#147dd9" } ] },
+        { "featureType": "poi", "elementType": "geometry.fill", "stylers": [ { "color": "#d3ffcc" } ] },{ "elementType": "labels", "stylers": [ { "visibility": "off" } ] } 
+      ];
     self.map = new google.maps.Map(document.getElementById('map'), {
       center: {
         lat: 50.0830986,
         lng: 6.7613762
       },
       zoom: 8,
       mapTypeId: 'roadmap',
-      styles: [ 
-        { "featureType": "road", "elementType": "geometry.fill", "stylers": [ { "color": "#4f9f92" }, { "visibility": "on" } ] },
-        { "featureType": "water", "elementType": "geometry.stroke", "stylers": [ { "color": "#feff95" }, { "visibility": "on" }, { "weight": 1.2 } ] },
-        { "featureType": "landscape", "elementType": "geometry", "stylers": [ { "color": "#adff9d" }, { "visibility": "on" } ] },
-        { "featureType": "water", "stylers": [ { "visibility": "on" }, { "color": "#147dd9" } ] },
-        { "featureType": "poi", "elementType": "geometry.fill", "stylers": [ { "color": "#d3ffcc" } ] },{ "elementType": "labels", "stylers": [ { "visibility": "off" } ] } 
-      ]
+      styles: ((self.settings.usePokemonGoMapStyle || self.settings.usePokemonGoMapStyle == undefined) ? mapStyle1 : []) // disable Pokemon Go map style only when explicitly instructed to
```

Comment:
Last time I checked, most users would rather use the Pokemon Go style. So I intentionally made it use the style when the setting doesn't exist.